static/app/components/smartSearchBar/index.tsx

```diff
@@ -377,6 +380,10 @@ class SmartSearchBar extends Component<DefaultProps & Props, State> {
       // eslint-disable-next-line react/no-did-update-set-state
       this.setState(this.makeQueryState(addSpace(query ?? undefined)));
     }
+
+    if (_actionBarItems?.length !== actionBarItems?.length) {
```

Comment:
Prefixing with `_` generally means it's unused, do we need to do that?

Reply:
or sorry, is there a better name we can give it

Reply:
💯 renamed it to `lastActionBarItems` 👍